variety/Util.py

```diff
@@ -890,12 +897,67 @@ def copy_with_replace(from_path, to_path, search_replace_map):
         with open(to_path + ".partial", "w") as file:
             file.write(data)
             file.flush()
-        os.rename(to_path + ".partial", to_path)
+        shutil.move(to_path + ".partial", to_path)
 
     @staticmethod
     def get_exec_path():
         return os.path.abspath(sys.argv[0])
 
+    @staticmethod
+    def set_windows_registry_key(path, name, value, dtype=None, master='HKEY_CURRENT_USER'):
+        import winreg
+
+        # Type is only used if creating a new value
+        if not dtype or isinstance(value, str):
+            dtype = winreg.REG_SZ
+        elif isinstance(value, int):
+            dtype = winreg.REG_DWORD
+        else:
+            dtype = winreg.REG_SZ
```

Comment:
nit: we can probably just collapse this into one check for `isinstance(value, int)`?